docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,393 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
+                "precedence" : "default"
+            }
+        }
+    ]
+}
+'
+```
+
+This example doesn't show the health checks in this example in order to highlight the important change, namely the
+addition of the `listenOptions` section. The first terminator is set to `required` and the second is set to `default`.
+Should the health check for the primary endpoint fail, the terminator precedence will be dropped to `failed` and new
+traffic will start flowing to the fail-over server. Should the primary recover, the health check will detect this and
+the precedence will be reset to `required`.
+
+Note that in addition to precedence, cost may also be set in the `listenOptions`.
+
+### Standalone Tunneler Hosting
+Most of the above applies to standalone tunnelers as well. The primary difference is in placement. Generally a tunneler
+will be running on the same machine as the application server. This means that there would be two tunnelers running, one on
+each of the hosts. The configuration could then reference `localhost`, with only a single terminator
+in the host config. The configuration would looking something like the following:
+
+:::note
+Not all tunneler implementations have support for the full `host.v1` and `host.v2` configs. The Go based tunneler,
+which is included in the edge router, has full support. Support in tunnelers for other languages may be partial,
+or non-existent.
+:::
+
+```
+ziti edge update config test-host-config host.v2 --data '
+{
+    "terminators" : [
+        {
+            "address": "localhost",
+            "port" : 8080,
+            "protocol": "tcp",
+            "portChecks" : [
+                {
+                    "address" : "localhost:8080",
+                    "interval" : "5s",
+                    "timeout" : "100ms",
+                    "actions" : [
+                        {
+                            "trigger" : "fail",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark unhealthy"
+                        },
+                        {
+                            "trigger" : "pass",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark healthy"
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+}
+'
+```
+
+For fail-over setups, the precedence would be set on the identity, rather than in the configuration.
+
+### SDK Hosted
+
+SDK hosted applications do not require any configs. When they bind a service, a terminator is created on their behalf.
+The SDKs have controls allowing cost and precedence to be set from the hosting application. Finally, the connection to
+the edge router acts as a built in health check. If the SDK loses its connection to the edge router, the edge router will
+remove any associated terminators. When the SDK reconnects, it will re-bind and a new terminator will be established.
+
+### Other Health Check Options
```

Comment:
I feel like the "Other Health Check Options" section should probably just be removed. I don't think it's adding enough value